source/triggers/disable.txt

```diff
@@ -103,3 +111,26 @@ importing an application directory with {+cli-bin+}.
 
                realm-cli push
 
+
+Restoring from a Snapshot
+-------------------------
+
+Consider the following scenario:
+
+1. A database trigger is disabled or suspended.
+
+#. New documents are added while the trigger is disabled.
+
+#. The database is restored from a snapshot to a time prior to the new documents 
+   being added.
+
+#. The database trigger is restarted.
+
+In this case, the trigger will pick up all of the new file additions and fire a 
+notification for each. It will not resend prior notifications. 
+
+.. note::
+
+   If a database trigger is enabled during snapshot restoration, you will see an 
+   error in the Edit Trigger section of the Atlas UI because it cannot connect to 
+   the Atlas cluster during the restore process. 
```

Comment:
Will this resolve itself or should we add guidance like "wait for the snapshot restoration to complete and then try to enable the trigger"

Reply:
Just checked the ticket, I think I misread this and we could clear it up. I read "is enabled" as "if you re-enable a disabled trigger".

Do you think we could make it clearer that an already enabled trigger will remain enabled but won't be able to connect until the snapshot restore is complete?